recipe/meta.yaml

```diff
@@ -27,10 +19,7 @@ requirements:
     - make   # [unix]
     # This breaks a dependency cycle:
     # curl->libssh2->cmake->curl
-    - cmake-binary  # [not (osx and arm64)]
```

Comment:
do not remove thos cmake packages.  They are there to allow building without cyclic dependency in cmake.

Reply:
This is fulfilled by cmake-no-system, which is available for all architectures.

Reply:
thanks